docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,202 @@
+.. include:: ../../common_links.inc
+
+.. _ugrid model:
```

Comment:
Looking good.
But overall I wish we could say 'unstructured data' a bit more, and 'UGRID' a bit less.
The way I see it, we have borrowed UGRID as a template for our data model, enabling us to represent unstructured data.  But we aim to support unstructured data, per se, whatever form we got it in.  
So we want to make the point that this is (or at least will be) a general facility in Iris.
We kind-of proved that _already_, by interoperating with ESMF grids in esmf-regrid and VTK data in Geovista.

Reply:
77e3f95aa4bf4b57e1c60788fd0462191e6645ce